flink-yarn/src/main/scala/org/apache/flink/yarn/ApplicationMaster.scala

```diff
@@ -41,9 +41,9 @@ object ApplicationMaster{
   val MODIFIED_CONF_FILE = "flink-conf-modified.yaml"
 
   def main(args: Array[String]): Unit ={
-    val yarnClientUsername = System.getenv(Client.ENV_CLIENT_USERNAME)
-    LOG.info(s"YARN daemon runs as ${UserGroupInformation.getCurrentUser.getShortUserName} " +
-      s"' setting user to execute Flink ApplicationMaster/JobManager to ${yarnClientUsername}'")
+    val yarnClientUsername = System.getenv(FlinkYarnClient.ENV_CLIENT_USERNAME)
+    LOG.info(s"YARN daemon runs as ${UserGroupInformation.getCurrentUser.getShortUserName}" +
```

Comment:
I leave the scala string interpolation here. The message is only logged once, we have the info level on by default and it helps improving the clearness to distinguish between the two usernames.